src/controller/CHIPDeviceController.cpp

```diff
@@ -1444,8 +1449,9 @@ void DeviceCommissioner::OnSessionEstablishmentTimeout()
 
 void DeviceCommissioner::OnSessionEstablishmentTimeoutCallback(System::Layer * aLayer, void * aAppState)
 {
-    static_cast<DeviceCommissioner *>(aAppState)->OnSessionEstablishmentTimeout();
+    static_cast<DeviceCommissioner *>(aAppState)->OnSessionEstablishmentError(CHIP_ERROR_TIMEOUT);
```

Comment:
Why this change?

Reply:
I had a good reason for this a few weeks ago, but I can't remember why now....If I can't remember, I'll delete it for now.